android/src/main/java/com/instabug/reactlibrary/RNInstabugCrashReportingModule.java

```diff
@@ -79,41 +85,61 @@ public void run() {
      * Send handled JS error object
      *
      * @param exceptionObject Exception object to be sent to Instabug's servers
+     * @param userAttributes  (Optional) extra user attributes attached to the crash
+     * @param fingerprint     (Optional) key used to customize how crashes are grouped together
+     * @param levelType       different severity levels for errors
      */
     @ReactMethod
-    public void sendHandledJSCrash(final String exceptionObject) {
+    public void sendHandledJSCrash(final String exceptionObject, @androidx.annotation.Nullable ReadableMap userAttributes, @androidx.annotation.Nullable String fingerprint, String levelType) {
```

Comment:
Let's import `Nullable` instead of `androidx.annotation.Nullable` for readability.

We can also turn `level` into `Nullable` as well since we already default to level of error.

```suggestion
    public void sendHandledJSCrash(final String exceptionObject, @Nullable ReadableMap userAttributes, @Nullable String fingerprint, @Nullable String level) {
```